dev/tasks/build/start.frag

```diff
@@ -0,0 +1,13 @@
+( function ( root, factory ) {
+	// Register the CKEDITOR global.
+	root.CKEDITOR = factory();
```

Comment:
I thought about that... but just the fact that `require` is available in the page doesn't guarantee that developers will be using CKEditor in the AMD way. They could still go with the namespace way. For example when copying pasting code from documentation or other sources.

For instance, developers may not even be developing with AMD at all, but maybe their CMSs are making `require` available and they don't know it. They'll try to use the CKEDITOR global then and have troubles.

Therefore, it sounded like the safest approach to enable both ways.